client/my-sites/upgrades/checkout-thank-you/site-redirect-details.jsx

```diff
@@ -10,29 +10,38 @@ import { getDomainManagementUrl } from './utils';
 import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from 'components/purchase-detail';
 
-const SiteRedirectDetails = ( { selectedSite } ) => {
+const SiteRedirectDetails = ( { selectedSite, domain } ) => {
 	return (
 		<div>
 			<PurchaseDetail
 				icon="external"
-				title={ i18n.translate( 'Redirect now working' ) }
-				description={ i18n.translate( 'Visitors to your site will be redirected to your chosen target.' ) }
-				buttonText={ i18n.translate( 'Test Redirect' ) }
-				href={ selectedSite.URL }
+				title={ i18n.translate( 'Test the redirect' ) }
+				description={
+					i18n.translate(
+						'Visitors to your site will be automatically redirected to {{em}}%(url)s{{/em}}.',
+						{
+							args: { url: domain },
+							components: { em: <em /> }
+						}
+					)
+				}
+				buttonText={ i18n.translate( 'Try it now' ) }
+				href={ `http://${ selectedSite.wpcom_url }` }
 				target="_blank" />
 
 			<PurchaseDetail
 				icon="cog"
 				title={ i18n.translate( 'Change redirect settings' ) }
-				description={ i18n.translate( 'You can disable the redirect or change the target at any time.' ) }
-				buttonText={ i18n.translate( 'My Domains' ) }
-				href={ getDomainManagementUrl( selectedSite ) } />
+				description={ i18n.translate( 'Disable the redirect by choosing a different primary domain, or change the target address.' ) }
```

Comment:
In the doc this is "Disable the redirect by choosing a different primary domain, or change the target address, from the Domains screen."